modules/cart-magento/mappers/ToCartDiscount.ts

```diff
@@ -2,5 +2,5 @@ import ToMoney from '#ioc/mappers/ToMoney'
 
 export default (data: any) => ({
   amount: ToMoney(data.amount),
-  label: data.label as string,
+  label: (data.label !== ' ' && (data.label as string)) || ('' as string),
```

Comment:
![image](https://user-images.githubusercontent.com/3133155/215451791-8b3a6d34-3c5b-4912-bbec-2f315883a6b1.png)

Reply:
If this is something truly non-negotiable with BE, at least use .trim() string method.